lib/maze/client/selenium/bb_client.rb

```diff
@@ -4,7 +4,7 @@ module Selenium
       class BitBarClient < BaseClient
         def start_session
           config = Maze.config
-          if ENV['BUILDKITE']
+          if ENV['BUILDKITE'] && !Maze.config.aws_public_ip
```

Comment:
This could be in the `if` on line 19 to avoid checking for the public IP twice

Reply:
Yup, that's pretty nasty.